src/lib/components/layout/v2/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+import * as defaultTheme from '../../../style/theme';
+import { getTheme } from '../../../utils';
+
+const LayoutContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  box-sizing: border-box;
+`;
+
+const Navigation = styled.div`
+  height: ${defaultTheme.navbarHeight};
+`;
+
+export const ContextContainer = styled.div<{
+  background?: string;
+}>`
+  background: ${(props) =>
+    props.background || getTheme(props)['backgroundLevel2']};
```

Comment:
Context is not supposed to have any background by default I think. It should take the one from the LayoutContainer